atrium/models/AccountOwner.php

```diff
@@ -0,0 +1,79 @@
+<?php
```

Comment:
On second thought, are these classes neccessary? Since there is no behavior in either class, it might be easier to just keep the associative arrays and use them without creating an object.